assets/css/style.css

```diff
@@ -8,57 +8,45 @@
 
 
 
-.undrtxt{
+.text{
     padding-top: 40px;
     font-family: 'Times New Roman', Times, serif;
     font-size: 40px;
     color: black;
     text-align: center;
 }
-
+ 
 .dish{
     display: flex;
-    padding-top: 60px;
-}
- 
-.dsc{
-    width: 400px;
-    height: 400px;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    gap: 200px;
+     width: 400px;

```

Comment:
pls get rid of the px unit. You have to use em or % to make your markup displaying well for different screen. So let's do it for iPad mini at first
![image](https://user-images.githubusercontent.com/18435140/192056967-f04b7b29-9202-4b08-a23d-d5c06ec89f0e.png)

Reply:
Done